scripts/convert-databases.bash

```diff
@@ -0,0 +1,278 @@
+#!/usr/bin/env bash
+
+DEFAULT_DBCONV=/usr/local/bin/dbconv
+DEFAULT_SRC=/home/user/.arbitrum/arb1/nitro
+
+dbconv=$DEFAULT_DBCONV
+src=$DEFAULT_SRC
+dst=
+force=false
+skip_existing=false
+clean="failed"
+
+l2chaindata_status="not started"
+l2chaindata_ancient_status="not started"
+arbitrumdata_status="not started"
+wasm_status="not started"
+classicmsg_status="not started"
+
+checkMissingValue () {
+    if [[ $1 -eq 0 || $2 == -* ]]; then
+        echo "missing $3 argument value"
+        exit 1
+    fi
+}
+
+printStatus() {
+    echo "== Conversion status:"
+    echo "   l2chaindata database: $l2chaindata_status"
+    echo "   l2chaindata database freezer (ancient): $l2chaindata_ancient_status"
+    echo "   arbitrumdata database: $arbitrumdata_status"
+    echo "   wasm database: $wasm_status"
+    echo "   classic-msg database: $classicmsg_status"
+}
+
+printUsage() {
+echo Usage: $0 \[OPTIONS..\]
+    echo
+    echo OPTIONS:
+    echo "--dbconv          dbconv binary path (default: \"$DEFAULT_DBCONV\")"
+    echo "--src             directory containing source databases (default: \"$DEFAULT_SRC\")"
+    echo "--dst             destination directory"
+    echo "--force           remove destination directory if it exists"
+    echo "--skip-existing   skip convertion of databases which directories already exist in the destination directory"
+    echo "--clean           sets what should be removed in case of error, possible values:"
+	echo "                      \"failed\" - remove database which conversion failed (default)"
+    echo "                      \"none\"   - remove nothing, leave unfinished and potentially corrupted databases"
+    echo "                      \"all\"    - remove whole destination directory"
+}
+
+removeDir() {
+    cmd="rm -r \"$1\""
+    echo $cmd
+    eval $cmd
```

Comment:
not need to use eval here, you can call `rm` directly

Reply:
it's needed to support directories with spaces in name. Otherwise I get:
```
== Note: removing only failed destination directory
rm -r "dstdb with spaces/l2chaindata"
rm: "dstdb: No such file or directory
rm: with: No such file or directory
rm: spaces/l2chaindata": No such file or directory
```